src/modules/src/controller_lee_payload.c

```diff
@@ -188,6 +190,13 @@ static controllerLeePayload_t g_self = {
   .Kpos_I ={10, 10, 10},
   .Kpos_I_limit = 0,
 
+  // Payload attitude gains
+
+  .Kprot_P = {0.01, 0.01, 0.01},
+  .Kprot_P_limit = 100,
+  .Kprot_D = {0.005, 0.005, 0.005},
+  .Kprot_D_limit = 100,
```

Comment:
We need new params, so we can tune it without reflashing the firmware

Reply:
done